src/test/shell/bazel/bazel_coverage_cc_test_gcc.sh

```diff
@@ -875,6 +875,79 @@ EOF
   assert_contains 'name: "test.lcov"' bep.txt
 }
 
+function test_coverage_with_experimental_split_coverage_postprocessing_only() {
+  local -r LCOV=$(which lcov)
+  if [[ ! -x ${LCOV:-/usr/bin/lcov} ]]; then
+    echo "lcov not installed. Skipping test."
+    return
+  fi
+  cat << EOF > WORKSPACE
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+  name = "com_google_googletest",
+  urls = ["https://github.com/google/googletest/archive/5c08f92c881b666998a4f7852c3cf9e393bf33a7.zip"],
+  strip_prefix = "googletest-5c08f92c881b666998a4f7852c3cf9e393bf33a7",
+)
+
+http_archive(
+    name = "rules_cc",
+    urls = ["https://github.com/bazelbuild/rules_cc/releases/download/0.0.2/rules_cc-0.0.2.tar.gz"],
+    sha256 = "58bff40957ace85c2de21ebfc72e53ed3a0d33af8cc20abd0ceec55c63be7de2",
+)
+EOF
+
+  cat << EOF > BUILD
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+
+cc_library(
+    name = "hello-greet",
+    srcs = ["hello-greet.cc"],
+    hdrs = ["hello-greet.h"],
+)
+cc_test(
+  name = "hello-test",
+  srcs = ["hello-test.cc"],
+  deps = [
+    "@com_google_googletest//:gtest_main",
+    ":hello-greet",
+  ],
+)
+EOF
+
+  cat << EOF > hello-greet.h
+#ifndef LIB_HELLO_GREET_H_
+#define LIB_HELLO_GREET_H_
+
+#include <string>
+
+std::string get_greet(const std::string &thing);
+
+#endif
+EOF
+
+  cat << EOF > hello-greet.cc
+#include <string>
+
+std::string get_greet(const std::string& who) {
+  return "Hello " + who;
+}
+EOF
+
+  cat << EOF > hello-test.cc
+#include <gtest/gtest.h>
+#include "hello-greet.h"
+
+TEST(HelloTest, GetGreet) {
+  EXPECT_EQ(get_greet("Bazel"), "Hello Bazel");
```

Comment:
This could just be `return 0;` or something - the actual test result doesn't matter, and we don't want to try pulling in gtest.